File: JavaScript/exercicio-04/solucao.js
const prompt = require("prompt-sync")();
const readline = require("readline-sync");

function main() {

  function soma() {

    const num1 = parseInt(prompt("Digite um número: "));
    const num2 = parseInt(prompt("Digite um outro número: "));
    const soma = num1 + num2
  
    console.log(`A soma entre ${num1} e ${num2} é igual a ${soma}`);
  }
  
  soma();

}

main();
readline.question('Digite Enter para sair ... ');

File: C/exercicio-04/solucao.c
#include <stdio.h>

void somar() {
  int num1, num2, soma;

  printf("Digite um número: " );
  scanf("%d", &num1);

  printf("Digite um outro número: " );
  scanf("%d", &num2);

  soma = num1 + num2;

  printf("A soma de %d e %d é igual a %d", num1, num2, soma);
}

int main() {
  somar();
  return 0;
}